lib/jsdom/living/custom-elements/CustomElementRegistry-impl.js

```diff
@@ -61,9 +62,10 @@ class CustomElementRegistryImpl {
     this._globalObject = globalObject;
   }
 
-  // https://html.spec.whatwg.org/#dom-customelementregistry-define
-  define(name, ctor, options) {
+  // https://html.spec.whatwg.org/multipage/custom-elements.html#dom-customelementregistry-define
+  define(name, ctorWrapper, options) {
```

Comment:
Please don't make unrelated changes in the PR. You can file issues asking if they are OK but it is not good to do so unsolicited.

Reply:
If I didn’t do this, then it would have issues later, because the `CustomElementRegistry` implementation works with the `objectReference`, rather than with the `invokeTheCallbackFunction` wrapper.

Reply:
Renaming is not necessary.